zeppelin-interpreter/src/main/thrift/genthrift.sh

```diff
@@ -17,176 +17,25 @@
 # * limitations under the License.
 # */
 
+# Determine the base directory dynamically (directory where the script is located)
+THRIFT_EXEC="$(dirname "$(realpath "$0")")"
```

Comment:
@pan3793 It seems feasible to implement the configuration to operate independently of the working directory using THRIFT_EXEC="$(dirname "$(realpath "$0")")". This approach allows for determining the location of the thrift script file itself and executing the thrift operations from that location.

Reply:
I understand your approach. However, the path is not the Thrift.exe, but the Zeppelin Thrift folder. Please change the variable name, it has just confused me a lot.

Reply:
`THRIFT_EXEC` should point to the thrift command directly.

Reply:
why I prefer `THRIFT_EXEC` instead of `THRIFT_HOME`?

the `thrift` binary has no extra dependency and can be copied to any folder directly, for example, I put the thrift binaries under `$HOME/bin`

```
$ ll bin | grep thrift
lrwxr-xr-x@ 1 chengpan  staff    13B Aug 19 19:35 thrift -> thrift-0.16.0
-rwxr-xr-x@ 1 chengpan  staff   3.4M Aug 19 11:13 thrift-0.14.1
-r-xr-xr-x@ 1 chengpan  staff   3.3M Aug 19 18:35 thrift-0.16.0
-r-xr-xr-x@ 1 chengpan  staff   3.6M Aug  1 10:15 thrift-0.20.0
-rwxr-xr-x@ 1 chengpan  staff   2.9M Aug 19 11:12 thrift-0.9.3
```